NeuralNetwork/stdafx.h

```diff
@@ -5,11 +5,14 @@
 
 #pragma once
 
+#ifdef _WIN32
```

Comment:
Only include these things on windows (targetvar.h has headers missing on unix systems, and PCH's aren't usually used.)